### PR TITLE
Bootstrap default flist to use based on runmode

### PR DESCRIPTION
**Pull request by muhamadazmy**

### CODE REVIEW

File: bootstrap/etc/zinit/init/bootstrap.sh [hunk 1]
```diff
@@ -17,21 +15,48 @@ function retry() {
     done
 }
 
-# bootflist reads the boot flist name from kernel cmd
-# TODO: this should probably be not allowed at somepoint to
-function bootflist() {
+function param() {
```
Comment (maxux) [inline bootstrap/etc/zinit/init/bootstrap.sh]: `function` is deprecated

File: bootstrap/etc/zinit/init/bootstrap.sh [hunk 2]
```diff
@@ -17,21 +15,48 @@ function retry() {
...
             return 0
         fi
     done
 
-    echo ${DEFAULT_FLIST}
+    return 1
 }
 
-FLIST=$(bootflist)
+function default_param() {
```
Comment (maxux) [inline bootstrap/etc/zinit/init/bootstrap.sh]: `function` is deprecated

File: bootstrap/etc/zinit/init/bootstrap.sh [hunk 3]
```diff
@@ -17,21 +15,48 @@ function retry() {
     done
 }
 
-# bootflist reads the boot flist name from kernel cmd
-# TODO: this should probably be not allowed at somepoint to
-function bootflist() {
+function param() {
+    local KEY=$1
+    local KEY=$(echo $KEY=)
```
Comment (maxux) [inline bootstrap/etc/zinit/init/bootstrap.sh]: What about `local KEY="${1}="` ?

File: bootstrap/etc/zinit/init/bootstrap.sh (hunk 1 above)
Comment (muhamadazmy) [reply bootstrap/etc/zinit/init/bootstrap.sh]: Seriously ? I didn't know that

File: bootstrap/etc/zinit/init/bootstrap.sh (hunk 3 above)
Comment (muhamadazmy) [reply bootstrap/etc/zinit/init/bootstrap.sh]: Haha, i must have been low on coffee !

**Review by maxux — APPROVED** [review]

**Pull request merged**